tests/installation/select_patterns_and_packages.pm

```diff
@@ -185,11 +186,19 @@ sub run {
             die "looping for too long" unless ($counter--);
             my $needs_to_be_selected;
             my $ret = check_screen('on-pattern', 1);
+            my $current_pattern = 'UNKNOWN_PATTERN';
 
             if ($ret) {    # unneedled pattern
                 for my $wp (keys %wanted_patterns) {
```

Comment:
I guess we need to rename wanted_patterns to something else, as now we have unwanted patterns there as well. But I like the idea!

Reply:
Renamed to `%patterns`